imperium-mindustry/src/main/kotlin/com/xpdustry/imperium/mindustry/security/VoteKickCommand.kt

```diff
@@ -163,6 +163,7 @@ class VoteKickCommand(instances: InstanceManager) :
 
     override suspend fun onVoteSessionFailure(session: VoteManager.Session<Context>) {
         runMindustryThread { freezes.setTemporaryFreeze(session.objective.target, null) }
+        session.objective.target.sendMessage("[sky]You are involved in a vote kick so you will be temporarily frozen. Being frozen prevents you from building or interacting. You will be un-frozen once the vote kick ends."
```

Comment:
@phinner this works right?

Reply:
You are telling the target they are still frozen while :
- the line above disables the temporary freeze.
- `onVoteSessionFailure` runs only after a failed vote, so why still freezing ?

Reply:
Damn i looked at the wrong line i guess :( ill fix it!